docs/changelog.adoc

```diff
@@ -13,45 +13,20 @@
 
 *Tested Compatibility:*
 
-* Apache Cassandra 2.1.18, 2.2.10, 3.0.14, 3.11.0
-* Apache HBase 0.98.24-hadoop2, 1.2.6, 1.3.1
+* Apache Cassandra 2.1.20, 2.2.10, 3.0.14, 3.11.0
+* Apache HBase 1.2.6, 1.3.1, 1.4.4
 * Google Bigtable 1.0.0
```

Comment:
This should be: 1.0.0, 1.1.2, 1.2.0, 1.3.0, 1.4.0 as per https://github.com/JanusGraph/janusgraph/blob/master/docs/versions.adoc

Reply:
Updated. I think we should eliminate this duplication in the next release.